dev/TeachingTip/TeachingTip.cpp

```diff
@@ -783,18 +803,26 @@ void TeachingTip::OnTargetOffsetChanged()
 
 void TeachingTip::OnIsLightDismissEnabledChanged()
 {
-    if (m_popup)
-    {
-        m_popup.get().IsLightDismissEnabled(IsLightDismissEnabled());
-    }
-
     if (IsLightDismissEnabled())
     {
         winrt::VisualStateManager::GoToState(*this, L"LightDismiss"sv, false);
+        if (auto lightDismissIndicatorPopup = m_lightDismissIndicatorPopup.get())
+        {
+            lightDismissIndicatorPopup.IsLightDismissEnabled(true);
+            m_lightDismissIndicatorPopupClosedRevoker = lightDismissIndicatorPopup.Closed(winrt::auto_revoke, { this, &TeachingTip::OnLightDismissIndicatorPopupClosed });
+        }
     }
     else
     {
         winrt::VisualStateManager::GoToState(*this, L"NormalDismiss"sv, false);
+        if (auto lightDismissIndicatorPopup = m_lightDismissIndicatorPopup.get())
```

Comment:
With my new understanding of the tracker_ref issue, the recommendation for capturing into a local is "auto&&":
```suggestion
        if (auto&& lightDismissIndicatorPopup = m_lightDismissIndicatorPopup.get())
```